content/release-notes/2023-05-23-storage-and-compute.md

```diff
@@ -0,0 +1,9 @@
+---
+label: 'Storage'
+---
+
+### What's new
+
+- Compute: Updated PostgreSQL versions to 14.8 and 15.3, respectively.
+- Compute: Implemented a `cargo neon` utility to facilitate setting up the Neon project locally. [Setup instructions](https://github.com/neondatabase/neon#running-neon-database) have been updated to reflect this change.
+- Control Plane: Disabled the Write-Ahead Log (WAL) [recovery_prefetch](https://www.postgresql.org/docs/current/runtime-config-wal.html#GUC-RECOVERY-PREFETCH) setting, which controls whether to prefetch blocks referenced in the WAL during recovery. Prefetching is not required by Neon's hot standby implementation.
```

Comment:
it's also `compute` and it's needed for our replication implementation

Reply:
@vadim2404 Maybe I should leave this item out since it' for a feature that's not yet released. WDYT?

Reply:
it's kind of bug fix, but I'm ok with your suggestion